misc/protobuf-ballerina/src/main/java/org/ballerinalang/protobuf/cmd/GrpcCmd.java

```diff
@@ -161,50 +181,38 @@ public void execute() {
             outStream.println(msg.toString());
         }
         msg.append("Successfully generated ballerina file.").append(NEW_LINE_CHARACTER);
-        //delete temporary meta files
-        delete(new File("desc_gen"));
-        delete(new File("google"));
-        LOG.debug("Successfully deleted temporary files.");
+
         outStream.println(msg.toString());
     }
     
     /**
      * Generate the meta folder which needed for intermediate processing.
      *
-     * @param descriptorPath proto descriptor path
      * @return Temporary Created meta file.
      */
-    private File generateDescTempFolder(String descriptorPath) {
-        File descFile = new File(descriptorPath);
-        String path = descFile.getAbsolutePath().substring(0, descFile.getAbsolutePath()
-                .lastIndexOf(BalGenerationConstants.FILE_SEPARATOR));
-        File folderPath = new File(path);
-        try {
-            if (!folderPath.exists() && !folderPath.mkdirs()) {
-                throw new IllegalStateException("Couldn't create dir: " + descFile);
-            }
-            descriptorPath = descFile.getAbsolutePath();
-            byte data[] = new byte[0];
-            Path file = Paths.get(descriptorPath);
-            Files.write(file, data);
-        } catch (IOException e) {
-            throw new BalGenToolException("Error creating " + descriptorPath + " file.", e);
+    private File generateTempDirectory() {
+
```

Comment:
Extra new line?

Reply:
Fixed in f2bb0753488e0c357b61ea5a9d3639e43bdf9bfe